src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setNumber(new Random().nextInt(100));
+        ball.setColor(new ColorSupplier().getRandomColor());
+        System.out.print("Ball has a number: " + ball.getNumber() + " and color: " + ball.getColor());
```

Comment:
don't forget to remove System.out.print(ln)s before committing. We can see the result of how the code words in the main class (Application)